### PR TITLE
Feat/add GitHub support

### PR DESCRIPTION
**Pull request by jsdidierlaurent**

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -15,74 +15,77 @@ type (
...
-		Jenkins     map[string]*Jenkins     `json:"jenkins"`
-		AzureDevOps map[string]*AzureDevOps `json:"azureDevOps"`
+		Ping        map[string]*Ping
+		Port        map[string]*Port
+		HTTP        map[string]*HTTP
+		Pingdom     map[string]*Pingdom
+		TravisCI    map[string]*TravisCI
+		Jenkins     map[string]*Jenkins
+		AzureDevOps map[string]*AzureDevOps
+		Github      map[string]*Github
```
Comment (Alex-D) [inline config/config.go]: Alphabetical ordered?

File: config/config.go [hunk 2]
```diff
@@ -15,74 +15,77 @@ type (
...
-		Token   string `json:"token"`
+		URL     string
+		Timeout int // In Millisecond
+		Token   string
+	}
+
+	Github struct {
+		Timeout              int // In Millisecond
+		Token                string
+		IssueCacheExpiration int // In Millisecond
```
Comment (Alex-D) [inline config/config.go]: Can we support on-premise GitHub, with something like Pingdom `URL`

File: monitorable/config/usecase/usecase.go [hunk 3]
```diff
@@ -30,7 +31,7 @@ const (
 )
 
 var SupportedVersions = map[int]bool{
-	Version4: true,
+	Version5: true,
```
Comment (Alex-D) [inline monitorable/config/usecase/usecase.go]: Docs needs to be updated too

File: config/config.go (hunk 1 above)
Comment (jsdidierlaurent) [reply config/config.go]: Hum, so we need to change in everywere ?

**Pull request merged**